File: GLLara/GLLAmbientLightViewController.m
//
//  GLLAmbientLightViewController.m
//  GLLara
//
//  Created by Torsten Kammer on 25.02.13.
//  Copyright (c) 2013 Torsten Kammer. All rights reserved.
//

#import "GLLAmbientLightViewController.h"

@interface GLLAmbientLightViewController ()

@end

@implementation GLLAmbientLightViewController

- (id)init
{
    self = [super initWithNibName:@"GLLAmbientLightView" bundle:nil];
    if (self) {
        // Initialization code here.
    }
    
    return self;
}

- (IBAction)help:(id)sender;
{
	NSString *locBookName = [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleHelpBookName"];
	[[NSHelpManager sharedHelpManager] openHelpAnchor:@"ambientlight" inBook:locBookName];
}

@end

File: GLLara/GLLAmbientLightViewController.swift
//
//  GLLAmbientLightViewController.swift
//  GLLara
//
//  Created by Torsten Kammer on 23.07.22.
//  Copyright © 2022 Torsten Kammer. All rights reserved.
//

import Cocoa

@objc class GLLAmbientLightViewController: NSViewController {
    
    convenience init() {
        self.init(nibName: "GLLAmbientLightView", bundle: nil)
    }
    
    @IBAction override func showContextHelp(_ sender: Any?) {
        let locBookName = Bundle.main.object(forInfoDictionaryKey: "CFBundleHelpBookName") as! NSString?
        NSHelpManager.shared.openHelpAnchor("ambientlight", inBook: locBookName as NSHelpManager.BookName?)
    }
}
